test/Syntax/tokens_unknown_and_invalid.swift

```diff
@@ -0,0 +1,126 @@
+// To embed test byte sequence,
+// this source replace marker to byte sequence first in runtime.
+// Marker(N) have `ZN` style format. Z is Z, N is number.
+// Byte sequence is represented in escape sequence.
+// To avoid replace marker in sed command by sed itself,
+// marker is also represented in escape sequence.
+
+// RUN: cat %s | sed \
+
+// [0xC2] is utf8 2 byte character start byte.
+// 0xC2 without second byte is invalid UTF-8 sequence.
+// It becomes garbage text trivia.
+// Marker(1) is replaced to this sequence.
+
+// RUN: -e 's/'$(echo -ne "\x5a1")'/'$(echo -ne "\xc2")'/g' \
+
+// [0xCC, 0x82] in UTF-8 is U+0302.
+// This character is invalid for identifier start, but valid for identifier body.
+// It becomes unknown token.
+// If this type characters are conitguous, they are concatenated to one long unknown token.
+// Marker(2) is replaced to this sequence.
+
+// RUN: -e 's/'$(echo -ne "\x5a2")'/'$(echo -ne "\xcc\x82")'/g' \
+
+// [0xE2, 0x80, 0x9C] in UTF-8 is U+201C, left quote.
+// It becomes single character unknown token.
+// If this left quote and right quote enclosure text,
+// they become one long unknown token.
+// Marker(3) is replaced to this sequence.
+
+// RUN: -e 's/'$(echo -ne "\x5a3")'/'$(echo -ne "\xe2\x80\x9c")'/g' \
+
+// [0xE2, 0x80, 0x9D] in UTF-8 is U+201D, right quote.
+// It becomes single character unknown token.
+// Marker(4) is replaced to this sequence.
+
+// RUN: -e 's/'$(echo -ne "\x5a4")'/'$(echo -ne "\xe2\x80\x9d")'/g' \
+
+// [0xE1, 0x9A, 0x80] in UTF-8 is U+1680.
+// This character is invalid for swift source.
+// It becomes garbage trivia.
+// Marker(5) is replaced to this sequence.
+
+// RUN: -e 's/'$(echo -ne "\x5a5")'/'$(echo -ne "\xe1\x9a\x80")'/g' \
+
+// RUN: > %t
+
+// RUN: %swift-syntax-test -input-source-filename %t -dump-full-tokens 2>&1 | %FileCheck %t
```

Comment:
Could you consolidate `round_trip_unknown_and_invalid.swift` into this file? i.e.
```swift
// RUN: %swift-syntax-test -input-source-filename %t -dump-full-tokens 2>&1 | %FileCheck %t
// RUN: %round-trip-syntax-test --swift-syntax-test %swift-syntax-test --file %t
```

Reply:
OK!